nixos/modules/services/networking/unbound.nix

```diff
@@ -126,6 +126,8 @@ in
         ProtectSystem = true;
         ProtectHome = true;
         PrivateDevices = true;
+        Restart = "always";
```

Comment:
How about `Restart = "on-failure";`?

Reply:
Otherwise `unbound-control stop` would not work as expected.

Reply:
> How about Restart = "on-failure";?

Ok.
Then maybe also ```|| true``` at the end of ```ExecStopPost``` as it always fails

Reply:
`|| true` does not work without shell.
However adding `-` will work:
 
```
 ExecStopPost="-${pkgs.utillinux}/bin/umount ${stateDir}/dev/random";
```

Reply:
```unbound-control stop``` fails because of wrong path to conf file

```
# unbound-control stop
[1528844506] unbound-control[25592:0] error: Could not open /etc/unbound/unbound.conf: No such file or directory
[1528844506] unbound-control[25592:0] fatal error: could not read config file
```

supplying the path as cmdline argument does not help
```
# unbound-control -c /var/lib/unbound/unbound.conf stop
[1528844596] unbound-control[25596:0] warning: control-enable is 'no' in the config file.
error: Error setting up SSL_CTX client key and cert
139745239754432:error:02001002:system library:fopen:No such file or directory:bss_file.c:406:fopen('/var/lib/unbound/etc/unbound/unbound_control.pem','r')
139745239754432:error:20074002:BIO routines:FILE_CTRL:system lib:bss_file.c:408:
139745239754432:error:140DC002:SSL routines:SSL_CTX_use_certificate_chain_file:system lib:ssl_rsa.c:701:
```

it seems like bringing ```unbound-control``` to live requires some effort